flang/include/flang/Lower/AbstractConverter.h

```diff
@@ -86,6 +86,9 @@ class AbstractConverter {
   /// Get the code defined by a label
   virtual pft::Evaluation *lookupLabel(pft::Label label) = 0;
 
+  /// Create mlir instance for a symbol
+  virtual void createVar(const Fortran::semantics::Symbol &sym) = 0;
```

Comment:
This could be renamed as `createPrivatizedInstance`, if the only usage is for privitzation ?
Could be wrong here :), so I'll leave this to @schweitzpgi and others.